src/marker.rs

```diff
@@ -802,6 +821,20 @@ impl MarkerTree {
         }
     }
 
+    /// Checks if the requirement should be activated with the given set of extras without
+    /// evaluating the remaining environment markers
+    ///
+    /// Note that unlike [evaluate] this does not perform any checks for bogus expressions but will
+    /// simply return true. As caller you should separately perform a check with the an environment
```

Comment:
Nit: I think there's an extra word in "with the an environment"

Reply:
oops